### PR TITLE
jenkins-x 3.2.287 (new formula)

### PR DESCRIPTION
**Pull request by sanyer**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/jenkins-x.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  def install
+    # Don't dirty the git tree
+    rm_rf ".brew_home"
+
+    ENV.prepend_path "PATH", Formula["coreutils"].libexec/"gnubin" # needs GNU date
+    system "make", "build"
+    bin.install "build/jx"
+  end
+
+  test do
```
[the remark this answers is not in the document]
Comment (SMillerDev) [reply Formula/jenkins-x.rb:28]: Well, if it fails without a kubernetes cluster:
> If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
> Same if the software requires a virtual machine, docker instance, etc. to be running.

[the remark this answers is not in the document]
Comment (sanyer) [reply Formula/jenkins-x.rb:28]: Yeah, I was just thinking about this :-D

[the remark this answers is not in the document]
Comment (sanyer) [reply Formula/jenkins-x.rb:28]: @SMillerDev I've added a simple functionality test which fails.

**Comment by sanyer** [comment]
Any chance to review my latest changes, @SMillerDev ?

**Comment by SMillerDev** [comment]
Well, the build is still failing so that needs to be resolved first

**Pull request closed without merging**